sample-ios-app/src/iosCommonMain/kotlin/com.badoo.reaktive.sample.ios/ReaktiveSample.kt

```diff
@@ -0,0 +1,6 @@
+package com.badoo.reaktive.sample.ios
```

Comment:
Just noticed, you have wrong folder name. The name is "com.badoo.reaktive.sample.ios". Folders should be nested instead.